src/main/java/core/basesyntax/strategy/handler/impl/AddToStorage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.handler.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class AddToStorage {
+    public static void add(FruitTransaction transaction) {
```

Comment:
Let it be `non-static`.

Reply:
I can't make it non-static because I use this method in BalanceHandler and it gives me a compile error.

Reply:
Then you need to instantiate it in BalanceHandler)